runner/config.go

```diff
@@ -61,5 +62,14 @@ func ReadConfig(filePath string) (*Config, error) {
 		config.TestAutofix = meta.IsDefined("autofix")
 	}
 
+	for i, dir := range config.ExcludedDirs {
+		normalized, err := normalizeFilePath(dir)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
return for any failure? or should we log the error and continue (iterate further)?

Reply:
Returning because if the configuration parsing fails, analysis won't be able to continue further.

Reply:
Ok